frontend/src/static/js/components/webstatus-feature-badge.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright 2025 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {html, LitElement} from 'lit';
+import {customElement, state} from 'lit/decorators.js';
+import {BADGE_PARAMS_BY_TYPE} from '../utils/constants.js';
+
+export type BadgeType = 'css' | 'html' | 'interop';
```

Comment:
We can derive the keys from the object itself. See my comment below in the constants file.